truckersmp-cli

```diff
@@ -529,6 +531,27 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_current_steam_user():
+    """
+    Get the current AccountName with saved login credentials.
+
+    Returns the found steam login name or None.
```

Comment:
I'd like "if/otherwise" for conditions:
* If steam login name found, this returns ...
* Otherwise, this returns None.